tests/phpunit/includes/testcase-assets.php

```diff
@@ -0,0 +1,158 @@
+<?php
+require POLYLANG_DIR . '/include/api.php';
+
+class PLL_Assets_UnitTestCase extends PLL_UnitTestCase {
+	protected static $editor;
+	protected static $stylesheet;
+	protected $polylang_assets = array(
+		'header' => array(
+			'user' => 'source',
+		),
+		'footer' => array(
+			'pll_ajax_backend'   => 'name',
+			'polylang_admin-css' => 'name',
```

Comment:
There is a problem if we expect the CSS in the footer.

Reply:
After looking at it it seems we have to call `do_action( 'admin_print_styles' );` in the header to render the stylesheets, otherwise they are rendered in the footer thanks to `_wp_footer_scripts()` which is hooked on `admin_print_footer_scripts` and prints remaining stylesheets (see: https://github.com/WordPress/wordpress-develop/blob/6.0.1/src/wp-includes/script-loader.php#L2118-L2126). 
This explains why no error was raised.

Reply:
Fixed in c180077